TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/AddAlert/CheckAnswers.cshtml.cs

```diff
@@ -111,6 +113,7 @@ public override async Task OnPageHandlerExecutionAsync(PageHandlerExecutingConte
         AlertTypeName = JourneyInstance!.State.AlertTypeName;
         Details = JourneyInstance!.State.Details;
         Link = JourneyInstance!.State.Link;
+        LinkUri = TrsUriHelper.TryCreateWebsiteUri(Link, out var linkUri) ? linkUri : null;
```

Comment:
We could have a partner method to `Try...`, like `Uri CreateWebsiteUri(string uriString)` that throws if the link is invalid for use here.